java/Makefile

```diff
@@ -4,8 +4,9 @@ all: requirements version package
 
 requirements:
 	rm -rf build
-	mkdir build
+	mkdir -p build/googleapis-proto
 	cp pom.xml build/
```

Comment:
Are you sure this is correct? Please see below as this PR removes the `java/pom.xml` file!

Reply:
this is correct with file restoration